### ISSUE TITLE
jdbc: support cluster client as the transport

### ISSUE THREAD

**Issue by Totktonada** [tarantool/tarantool-java#199]

**Comment by Totktonada** [comment]
I guess this should be simpler now, after #163.

**Comment by nicktorwald** [comment]
Agreed. Because a Tarantool client is behind a SQLConnection, we need to elaborate on how a user can pass an initial set of Tarantool instances as well as parameters specific to the cluster config such as *discovery function* or *discovery delay*.
I'd consider doing that via a connection string like `jdbc:taramtool://host1:port1,host2:port2?clusterDiscoveryFunction=abc&clusterDiscoveryDelayMs=60000&otherParams`.

**Comment by Totktonada** [comment]
I think we should allow to pass all parameters (as usual as well as cluster ones) via properties and URI both.
